server/playerServer/scripts/PlayerDatabase.js

```diff
@@ -363,17 +363,154 @@ PlayerDatabase.prototype.resetProtectedRegions = function resetProtectedRegions
 	});
 };
 
+PlayerDatabase.prototype.removeProtectedRegion = function removeProtectedRegion (userid, room, regionId, callback) {
+	console.log("removeProtectedRegion", userid, room, regionId);
+	this.database.query("DELETE FROM regions WHERE owner = ? AND room = ? AND id = ?", [userid, room, regionId], function (err) {
+		if(err){
+			callback(err);
+			return;
+		}
+		this.database.query("delete from regions_permissions where regionId = ?", [regionId], function (err) {
+			callback(err);
+		}.bind(this));
+	}.bind(this));
+};
+
 PlayerDatabase.prototype.getProtectedRegions = function getProtectedRegions (room, callback) {
 	// TODO: Select permissions
-	this.database.query("SELECT owner, minX, minY, maxX, maxY FROM regions WHERE room = ?", [room], function (err, rows, fields) {
+	this.database.query("SELECT id, owner, minX, minY, maxX, maxY, minRepAllowed FROM regions WHERE room = ? ORDER BY id", [room], function (err, rows1, fields) {
```

Comment:
We might at some point select the protected regions without the permissions. So this should be split into 2 functions.